addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -184,7 +184,7 @@
                             <field name="depend_on_count" invisible="1" />
                             <field name="allow_task_dependencies" invisible="1" />
                             <field name="current_user_same_company_partner" invisible="1"/>
-                            <field name="tag_ids" context="{'project_id': project_id}" widget="many2many_tags" options="{'color_field': 'color', 'no_create': True, 'no_edit': True, 'no_edit_color': True}"/>
+                            <field name="tag_ids" context="{'project_id': project_id}" widget="many2many_tags" options="{'color_field': 'color', 'on_click_pipu': 'edit_color', 'no_create': True, 'no_edit': True}"/>
```

Comment:
this one is `"no_edit": true`. Can we edit the color in that case? If yes,should we?